lib/manager/tool-versions/extract.spec.ts

```diff
@@ -0,0 +1,158 @@
+import { Fixtures } from '../../../test/fixtures';
+import { extractPackageFile } from './extract';
+
+describe('manager/tool-versions/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('returns null for empty', () => {
+      expect(extractPackageFile('', '.tool-versions', {})).toBeNull();
+    });
+
+    it('extracts dependencies for supported tools', () => {
+      const res = extractPackageFile(
+        Fixtures.get('.tool-versions'),
+        'unused_file_name',
+        {}
+      );
+      expect(res).toEqual({
+        deps: [
+          {
+            currentValue: '10.15.0',
+            currentVersion: '10.15.0',
+            datasource: 'npm',
```

Comment:
This should reflect the datasource which it's actually using. I doubt it's `npm`

Reply:
Hi @rarkins I've replaced `npm` with `git-tags` since it seems to be more suitable here.